cmd/atlas-action/main.go

```diff
@@ -67,7 +67,7 @@ func (v VersionFlag) BeforeReset(app *kong.Kong) error {
 
 // RunActionCmd is a command to run one of the Atlas GitHub Actions.
 type RunActionCmd struct {
-	Action  string      `help:"Command to run" required:""`
+	Action  string      `help:"Command to run" required:"" env:"ATLAS_ACTION"`
```

Comment:
This change allows invoking the action via the env variable, which is convenient for BB case.
```shell
docker run -e CI=true -e ATLAS_ACTION=schema/push arigaio/atlas-action
```